src/pspm_sf.m

```diff
@@ -25,6 +25,7 @@
 %   │                   as 'dcm'
 %   │                   [cell_array] a cell array of methods mentioned above.
 %   ├─────────.filter:  filter settings; modality specific default
+%   ├────────.missing:  index of missing values to ignore
```

Comment:
this is not an index, it should be an epoch definition file.

Reply:
I will change it and use `get_timing` to load..
Just to confirm. I updated this line based on `pspm_sf_dcm` at line 32. Shall I make `.missing` to always refer to a missing epoch to all the functions?

Reply:
Sure. In this pull request the objective is to allocate .missing or .missing_data, thus I have enabled them.
For loading and calculating missing_data, I put them in pull request 484, where SF is following the design in DCM.